From c0c19a71c5481f3082afee2ac7962bd7abfcf134 Mon Sep 17 00:00:00 2001
From: Lukas Fleischer <info@cryptocrack.de>
Date: Sun, 22 Jan 2012 22:19:50 +0100
Subject: [PATCH] Makefile: Fix installation of "README" and "NEWS"

Initialize "INSTALL" in the top-level Makefile in order to avoid broken
install(1) invocations such as:

    dm0755 "/build/pkg/usr/share/doc/faketime/"
    /bin/sh: dm0755: command not found

Signed-off-by: Lukas Fleischer <info@cryptocrack.de>
---
 Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index f90e18e..d0225ba 100644
--- a/Makefile
+++ b/Makefile
@@ -1,3 +1,5 @@
+INSTALL = install
+
 all:
 	$(MAKE) -C src all
 	$(MAKE) -C test all
-- 
1.7.8.4

